robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commands.trapAmpArm;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class TrapAmpArmZeroCommand extends Command {
+
+  public TrapAmpArmZeroCommand() {
+    addRequirements(Robot.trapAmpArm);
+  }
+
+  @Override
+  public void initialize() {
+    Robot.trapAmpArm.set(Constants.TRAP_AMP_ARM.ZERO_SPEED);
+  }
+
+  @Override
+  public boolean isFinished() {
+    return Robot.trapAmpArm.getMotorVelocity() <= Constants.TRAP_AMP_ARM.ZERO_SPEED_STOP_TOLERANCE;
```

Comment:
You need an end() block to set the rotations count to zero

Reply:
And stop the motors, that is also important.

Reply:
i forgot about the end block completly